Source/WebInspectorUI/UserInterface/Models/CSSKeywordCompletions.js

```diff
@@ -1383,6 +1386,15 @@ WI.CSSKeywordCompletions._propertyKeywordMap = {
     "text-underline-width": [
         "normal", "medium", "auto", "thick", "thin", "calc()",
     ],
+    "text-wrap": [
+        "wrap", "nowrap", "balance", "stable", 
```

Comment:
are these not provided to the frontend by `CSS.getSupportedCSSProperties`?

ever since [204647@main](https://commits.webkit.org/204647@main) the backend should be including as much of this kind of info as possible, with the hardcoded list in `WI.CSSKeywordCompletions` being more of a legacy thing, so if these are included in `CSS.getSupportedCSSProperties` then id prefer we not add them as it's unnecessary

Reply:
That's right! The supported CSS property values are returned by `CSS.getSupportedCSSProperties`.

<img width="251" alt="Screenshot 2024-09-19 at 19 20 46" src="https://github.com/user-attachments/assets/9688776c-2496-45bb-adfc-5ef4059573f7">

I think I got confused by the missing `pretty` value because the feature was not enabled on my build. 

@Ahmad-S792 I apologize for missing this and asking you for this PR.

Reply:
No worries at all. Closing this now.